src/ZfcUser/Authentication/Adapter/Db.php

```diff
@@ -12,7 +12,7 @@
 use ZfcUser\Mapper\User as UserMapperInterface;
 use ZfcUser\Options\AuthenticationOptionsInterface;
 
-class Db extends AbstractAdapter implements ServiceManagerAwareInterface
```

Comment:
@svycka Then it should be marked for a new minor release at the least, and potentially a new major release. However, making code _forwards_ compatible, and removing deprecation notices, is generally considered reasonable for a minor release. While you typically don't want to remove features, if doing so provides greater compatibility, it's the responsible thing to do.